python_src/source/simulator/simulator.py

```diff
@@ -1,12 +1,48 @@
 import lgsvl
+import os
 
 
 class Simulator:
-    def __init__(self):
-        # Connect to the simulator. Load the scene from http://localhost:8080/#/Maps
-        sim = lgsvl.Simulator(address="localhost", port=8181)
-        if sim.current_scene == "SingleLaneRoad":
-            sim.reset()
-        else:
-            sim.load("SingleLaneRoad", seed=650387)
-        self.sim = sim
\ No newline at end of file
+    def __init__(self, seconds=30, scene="SingleLaneRoad", error_message=None, load_scene=True):
+        if error_message is None:
+            error_message = 'test timed out after {}s.'.format(seconds)
+        self.seconds = seconds
+        self.error_message = error_message
+        self.scene = scene
+        self.load_scene = load_scene
+
+    def __enter__(self):
```

Comment:
Using `__enter__` and `__exit__` for starting/stopping the simulation is a good idea. I may adopt that. 👍🏻